pxr/imaging/plugin/hdRpr/renderBuffer.cpp

```diff
@@ -161,11 +167,35 @@ VtValue HdRprRenderBuffer::GetResource(bool multiSampled) const {
         }
 
         VtDictionary dictionary;
+#ifdef HDRPR_ENABLE_VULKAN_INTEROP_SUPPORT
+        if (multiSampled) {
```

Comment:
Please leave comment about multiSampled:
// We are using multiSampled parameter as workaround since USD also calls this method, but with 'false' value

Reply:
Added the comment